sql/2019/17_CDN/17_05.sql

```diff
@@ -0,0 +1,9 @@
+#standardSQL
+# 17_05: Percentage of responses with strict-transport-security header
+SELECT
+  _TABLE_SUFFIX AS client,
+  ROUND(SUM(IF(LOWER(respOtherHeaders) LIKE "%strict-transport-security%", 1, 0)) / COUNT(0), 4) AS pctStrictTransportSecurity

```

Comment:
```suggestion
  ROUND(COUNTIF(LOWER(respOtherHeaders) LIKE "%strict-transport-security%") * 100 / COUNT(0), 2) AS pct
```

Reply:
SUM(IF()) changed to COUNTIF() in all 17_5, 6, 7, 8, 9, 10 and 11